packages/engine/src/transform/systems/TransformSystem.ts

```diff
@@ -402,13 +402,13 @@ const execute = () => {
   _projScreenMatrix.multiplyMatrices(camera.projectionMatrix, camera.matrixWorldInverse)
   _frustum.setFromProjectionMatrix(_projScreenMatrix)
 
-  for (const entity of frustumCulledQuery())
-    FrustumCullCameraComponent.isCulled[entity] = _frustum.containsPoint(
-      getComponent(entity, TransformComponent).position
-    )
-      ? 0
-      : 1
-
+  for (const entity of frustumCulledQuery()) {
```

Comment:
we might want to benchmark if having two queries here, one with bbox and one without gives better results, since we have a few extra conditions when they are combined

Reply:
Yes! I was thinking about this earlier, so I've been taking performance profiles for a while and I have seen little change, roughly ~3.5ms regardless of the extra bbox condition, and no discernable difference in fps.
![image](https://github.com/EtherealEngine/etherealengine/assets/102248647/c0e51d10-5001-46d2-b19c-50b8776dc9cb)

Reply:
![image](https://github.com/EtherealEngine/etherealengine/assets/102248647/11b11911-cda5-4698-95e5-ce50e9f5ecc8)

Looks like it does tack on some ms to get the bounding box component and then perform tests. In this case it looks like an extra 3ms for 128 avatars. I'm not a fan.